### PR TITLE
web: make ExternalServiceEditPage look like ExternalServicePage.

### PR DESCRIPTION
**Pull request by sashaostrikov**
This includes adding breadcrumbs and removing large code host kind icon panel. Also, "cancel" buttons are added at the top of the page and near "update" button at the bottom of the form.

Test plan:
Local sg run and manual tests.

### Quick demo

https://user-images.githubusercontent.com/94846361/212309347-628d4add-4c10-403d-aee7-2c31f1e680c0.mp4

## App preview:

- [Web](https://sg-web-ao-ui-ext-svc-page-improvements.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/components/externalServices/ExternalServiceEditPage.tsx [hunk 1]
```diff
@@ -123,19 +124,49 @@ export const ExternalServiceEditPage: React.FunctionComponent<React.PropsWithChi
...
-                        </div>
-                    )}
+                    <PageHeader
+                        path={[
+                            { icon: mdiCog },
+                            { to: '/site-admin/external-services', text: 'Code hosts' },
+                            {
+                                text: (
+                                    <>
+                                        {externalService.displayName}{' '}
```
Comment (mrnugget) [inline client/web/src/components/externalServices/ExternalServiceEditPage.tsx:138]: Nit: shouldn't the `{' '}` go into the `externalServiceCategory &&`?

Edit: or can you instead give the `Icon` and `ml-2`?

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.02% (+3.19 kb) | 0.03% (+3.19 kb) | -0.14% (-1) 🔽 |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ao/ui/ext-svc-page-improvements/compare-report.html;tab=live_object) for a full comparison between the commits c8eea64 and 35af299 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/components/externalServices/ExternalServiceEditPage.tsx [hunk 2]
```diff
@@ -123,19 +124,49 @@ export const ExternalServiceEditPage: React.FunctionComponent<React.PropsWithChi
...
+                                        {externalServiceCategory && (
+                                            <Icon
+                                                inline={true}
+                                                as={externalServiceCategory.icon}
+                                                aria-label="Code host logo"
+                                                className="mr-2"
```
Comment (mrnugget) [inline client/web/src/components/externalServices/ExternalServiceEditPage.tsx:144]: Why `mr-2`?

**Review by mrnugget — APPROVED** [review]
Love it!

File: client/web/src/components/externalServices/ExternalServiceEditPage.tsx (hunk 1 above)
Comment (sashaostrikov) [reply client/web/src/components/externalServices/ExternalServiceEditPage.tsx:138]: oh man, `ml-2` for sure 🤦🏻

File: client/web/src/components/externalServices/ExternalServiceEditPage.tsx (hunk 2 above)
Comment (sashaostrikov) [reply client/web/src/components/externalServices/ExternalServiceEditPage.tsx:144]: accident, fixed!

**Pull request merged**